lib/vye/dgib/authentication_token_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Vye
+  module DGIB
+    class AuthenticationTokenService
+      ALGORITHM_TYPE = 'RS256'
```

Comment:
Should this be RSA256?

Reply:
My understanding is that RS256 is the valid algorithm_type for jwt (RSA using SHA256)